pkg/kubectl/describe.go

```diff
@@ -953,6 +960,56 @@ func describeNode(node *api.Node, pods []*api.Pod, events *api.EventList) (strin
 	})
 }
 
+// HorizontalPodAutoscalerDescriber generates information about a horizontal pod autoscaler.
+type HorizontalPodAutoscalerDescriber struct {
+	client       *client.Client
+	experimental *client.ExperimentalClient
+}
+
+func (d *HorizontalPodAutoscalerDescriber) Describe(namespace, name string) (string, error) {
+	hpa, err := d.experimental.HorizontalPodAutoscalers(namespace).Get(name)
+	if err != nil {
+		return "", err
+	}
+	return tabbedString(func(out io.Writer) error {
+		fmt.Fprintf(out, "Name:\t%s\n", hpa.Name)
+		fmt.Fprintf(out, "Namespace:\t%s\n", hpa.Namespace)
+		fmt.Fprintf(out, "Labels:\t%s\n", formatLabels(hpa.Labels))
+		fmt.Fprintf(out, "CreationTimestamp:\t%s\n", hpa.CreationTimestamp.Time.Format(time.RFC1123Z))
+		fmt.Fprintf(out, "Reference:\t%s\\%s\\%s\\%s\n",
```

Comment:
Use forward slashes for consistency (I don't think we use a backslash anywhere else in the code)